apple/Handlers/RNPanHandler.m

```diff
@@ -102,6 +128,11 @@ - (void)interactionsBegan:(NSSet *)touches withEvent:(UIEvent *)event
   } else {
     super.minimumNumberOfTouches = _realMinimumNumberOfTouches;
   }
+
+  // We don't want to send stylusData object if stylus is not used. Therefore we set pressure to -1.
+  // If it stays at this value, it means that stylus was not used and we can omit sending stylus data.
+  _stylusData.pressure = -1;
```

Comment:
Shouldn't this be in reset?

Reply:
Done in [252a7de](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/252a7de9500c43e4eae95af15714af28dcf37033)